mempool/src/shared_mempool/tasks.rs

```diff
@@ -585,10 +458,3 @@ pub(crate) async fn process_config_update<V>(
         .restart(config_update)
         .expect("failed to restart VM validator");
 }
-
-/// creates uniques request id for the batch in the format "{start_id}_{end_id}"
-/// where start is an id in timeline index  that is lower than the first txn in a batch
-/// and end equals to timeline ID of last transaction in a batch
-fn create_request_id(start_timeline_id: u64, end_timeline_id: u64) -> String {
-    format!("{}_{}", start_timeline_id, end_timeline_id)
-}
```

Comment:
so per the spec, the request_id can be any String, it's just this particular implementation chooses to store the timeline ID range into the request_id. This assumption only applies to the broadcasts this node crafts and (correctly) does not make this assumption about incoming broadcasts from other peers - it only sends the ACK back with the whatever request_id String value it was provided. The spec requirement of the request_id is set to a generic string (vs some more structured format like u64, u64) because it should be up to the implementor to choose how to manage their broadcasts, and the concept of timeline ID is not part of the spec, so I hesitate to expose that into the spec.. lmk your thoughts